### DISCUSSION TITLE
Wonder why there are two translations of "Falkland Islands"?

### DISCUSSION THREAD

**Discussion by sh-cho** [SmileyChris/django-countries#371] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/11611397/132638311-8e9884e9-3859-45a9-b07f-b66055592bd1.png)

As far as I understand, base.py has some old names of countries (Like "SZ" -> old: "Swaziland", new: "Eswatini") but for "FK", old name is "Falkland Islands  [Malvinas]", and new name is "Falkland Islands (Malvinas)". I just wonder why.

https://github.com/SmileyChris/django-countries/blob/62ed634f2e96ea734416c07f4034f4f37b13e706/django_countries/base.py#L32

https://github.com/SmileyChris/django-countries/blob/62ed634f2e96ea734416c07f4034f4f37b13e706/django_countries/data.py#L104

**Comment by SmileyChris** [comment, marked as answer]
The one in base.py is a historic name that some older translations used before the english name in data.py was updated.
